docs/src/examples/components/Loader/Usage/LoaderExampleDelay.tsx

```diff
@@ -0,0 +1,8 @@
+import { Loader } from '@stardust-ui/react'
+import * as React from 'react'
+
+const LoaderExampleDelay: React.FC<{ knobs: { mounted: boolean } }> = ({ knobs }) => (
+  <div style={{ minHeight: 24 }}>{knobs.mounted && <Loader delay={500} />}</div>
```

Comment:
Maybe use here Flex with padding instead of div?

Reply:
Actually, I don't think that it's a case for `Flex`. We require there `minHeight` because `Loader` will render `null` with `delay`, the same behavior as there: https://evergreen.segment.com/components/spinner/

Reply:
I want to avoid this effect:

![loader-delay](https://user-images.githubusercontent.com/14183168/53483749-d716bc00-3a8a-11e9-8ff8-78553c31fe15.gif)

Reply:
Aha, got it! Let's leave it as it is then..